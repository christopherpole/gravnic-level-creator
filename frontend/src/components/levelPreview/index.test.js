import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import { LevelPreview } from './index';

configure({ adapter: new Adapter() });

describe('<LevelPreview />', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  it('Renders without exploding', () => {
    const levelPreview = shallow(<LevelPreview {...props} />);

    expect(levelPreview).toHaveLength(1);
  });

  it('Matches the current snapshot', () => {
    const levelPreview = shallow(<LevelPreview {...props} />);

    expect(toJson(levelPreview)).toMatchSnapshot();
  });
});
