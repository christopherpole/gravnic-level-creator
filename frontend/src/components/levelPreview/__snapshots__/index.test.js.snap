// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The tile selector Matches the current snapshot 1`] = `
<styled.div
  id="level-preview"
>
  <styled.div>
    <Connect(Component) />
  </styled.div>
</styled.div>
`;
